syntax = "proto3";

option java_package = "zemberek.proto";
option java_multiple_files = true;

package zemberek.normalization;

message NormalizationRequest {
  string input = 1;
}

message NormalizationResponse {
  string normalized_input = 2;
  // TODO(mdakin): Add structured result.
}

service NormalizationService {
  rpc Normalize (NormalizationRequest) returns (NormalizationResponse);
}